adoc-mode.el

```diff
@@ -540,8 +570,452 @@ To become a customizable variable when regexps for list items become customizabl
+  "Face for blockquote sections."
+  :group 'adoc-faces)
+(defvar adoc-blockquote-face 'adoc-blockquote-face)
+
+(defface adoc-code-face
+  '((t (:inherit fixed-pitch)))
+  "Face for inline code and fenced code blocks.
+  This may be used, for example, to add a contrasting background to
+  inline code fragments and code blocks."
+  :group 'adoc-faces)
+(defvar adoc-code-face 'adoc-code-face)
+
+(defface adoc-command-face
+  '((default (:inherit adoc-meta-face
+                       :weight bold
+                       :box(:line-width 2 :style released-button)))
+    (((background light))(:background "#f5f5f5" :foreground "black" :box (:color "#dcdcdc")))
+    (((background dark))(:background "#272822" :foreground "#e6db74" :box (:color "#e6db74"))))
+  "Face for command names."
+  :group 'adoc-faces)
+(defvar adoc-command-face 'adoc-command-face)
+
+(defface adoc-complex-replacement-face
+  '((default (:inherit adoc-meta-face
+                       :box (:line-width 2 :style released-button)))
+    (((background light)) (:background "plum1" :foreground "purple3" :box (:color "plum1")))
+    (((background dark)) (:background "purple3" :foreground "plum1" :box (:color "purple3"))))
+  "Markup that is replaced by something complex.
+  For example an image, or a table of contents.
+  AsciiDoc: image:...[...]"
+  :group 'adoc-faces)
+(defvar adoc-complex-replacement-face 'adoc-complex-replacement-face)
+
+(defface adoc-inline-code-face
+  '((t (:inherit (adoc-code-face font-lock-constant-face))))
+  "Face for inline code."
+  :group 'adoc-faces)
+(defvar adoc-inline-code-face 'adoc-inline-code-face)
+
+(defface adoc-passthrough-face
+  '((t :inherit (fixed-pitch adoc-gen-face)))
+  "For text that is passed through yet another marser/renderer.
+
+  Since this text is passed to an arbitrary renderer, it is unknown
+  wich of its chars are meta characters and which are literal characters."
+  :group 'adoc-faces)
+(defvar adoc-passthrough-face 'adoc-passthrough-face)
+
+(defface adoc-verbatim-face
+  '((((background light))
+     (:background "cornsilk"))
+    (((background dark))
+     (:background "saddlebrown")))
+  "For verbatim text.
+
+Verbatim in a sense that all its characters are to be taken
+literally. Note that doesn't necessarily mean that that it is in
+a typewritter font.
+For example 'foo' in the following examples. In parantheses is a
+summary what the command is for according to the given markup
+language.
+`foo`     (verbatim and typewriter font)
++++foo+++ (only verbatim)"
+  :group 'adoc-faces)
+(defvar adoc-verbatim-face 'adoc-verbatim-face)
+
+(defface adoc-pre-face
+  '((t (:inherit (adoc-code-face font-lock-constant-face))))
+  "Face for preformatted text."
+  :group 'adoc-faces)
+(defvar adoc-pre-face 'adoc-pre-face)
+
+(defface adoc-table-face
+  '((t (:inherit (adoc-code-face))))
+  "Face for tables."
+  :group 'adoc-faces)
+(defvar adoc-table-face 'adoc-table-face)
+
+(defface adoc-language-keyword-face
```

Comment:
Hmm, I don't see any usages of this face and I assume of some other faces as well. Were those copied from markdown-mode?

I think it'd be best to limit this PR just to things that are actually used right now and do any additions/improvements subsequently in other PRs.

Reply:
All unused faces removed. Some existing faces now inherits from faces from `font-lock.el`.